examples/src/main/scala/zio/logging/Examples.scala

```diff
@@ -13,19 +13,20 @@ object Examples extends zio.App {
   val logFormat = "[correlation-id = %s] %s"
 
   val env =
-    //Slf4jLogger.make((context, message) => logFormat.format(context(correlationId), message))
+    //Slf4jLogger.make((context, message) => logFormat.format(context(correlationId), message)) >>> Logging.withRootLoggerName("zio-logging-example")
     Logging.console(format =
       LogFormat.ColoredLogFormat((context, message) => logFormat.format(context(correlationId), message))
-    )
+    ) >>> Logging.modifyLogger(_.derive(correlationId("0000"))) >>> Logging.withRootLoggerName("root-logger")
 
+  import zio.duration._
   override def run(args: List[String]) =
     (for {
       fiber <- log.locally(correlationId("1234"))(ZIO.unit).fork
       _     <- log.info("info message without correlation id")
```

Comment:
Is this still without a correlation ID? In the line 19 you modify a logger and I would assume it will initially have correlationId 0000 applied.

Reply:
yes, that correct

Reply:
What exactly? That there will be correlationID or not?

Reply:
I see what you mean - message is misleading. in this line you will have correlation id from initial value.

Reply:
I changed message